update_prevalence.py

```diff
@@ -1399,12 +1405,12 @@ def process_regional_case_reports():
                     min_test_count = (
                         report.cumulative_testing
                         if min_test_count is None
-                        else min(min_test_count, report.cumulative_testing)
+                        else min(min_test_count, report.cumulative_testing) # type: ignore
                     )
                     max_test_count = (
                         report.cumulative_testing
                         if max_test_count is None
-                        else max(max_test_count, report.cumulative_testing)
+                        else max(max_test_count, report.cumulative_testing) # type: ignore
```

Comment:
In the above two cases, mypy wasn't smart enough to recognize that in this else statement, the first argument can never be None.